arangodb-net-standard.Test/UserApi/UserApiClientTest.cs

```diff
@@ -198,6 +198,22 @@ public async Task GetUserAsync_ShouldThrow_WhenUserDoesNotExist()
             Assert.Equal(1703, ex.ApiError.ErrorNum); // ERROR_USER_NOT_FOUND
         }
 
+        [Fact]
+        public async Task GetUsersAsync_ShouldSucceed()
+        {
+            GetUsersResponse response = await _userClient.GetUsersAsync();
+
+            Assert.False(response.Error);
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.NotNull(response.Result);
+            Assert.True(response.Result.Count() > 0);
+
+            foreach (AvailableUser user in response.Result)
+            {
+                Assert.False(string.IsNullOrEmpty(user.User));
```

Comment:
We create and delete users in the tests, so checking all users for specific values may not be reliable. However it might be worth checking that the user we are using appear in the list.

The check for not null or empty is to ensure we have values, then we trust that ArangoDB is working as expected to give us the values we need.

Reply:
I added a check for one of the fixture users.